flocker/common/test/test_interface.py

```diff
@@ -0,0 +1,133 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common.interface_decorator``.
```

Comment:
:question: Nit picking, but this test module will have "Tests  for ``flocker.common._interface``" not just this new function.